src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -627,7 +627,7 @@ private unsafe static (uint? userId, uint? groupId) GetUserAndGroupIds(string us
             // First try with a buffer that should suffice for 99% of cases.
             // Note: on CentOS/RedHat 7.1 systems, getpwnam_r returns 'user not found' if the buffer is too small
             // see https://bugs.centos.org/view.php?id=7324
-            const int BufLen = 1024;
+            const int BufLen = Interop.Sys.Passwd.InitialBufferSize4;
```

Comment:
Nit: looks like there's a straggling `4` at the end erroneously

Reply:
Uggh, staging commits fails for me again.  Will fix